src/metabase/query_permissions/core.clj

```diff
@@ -0,0 +1,17 @@
+(ns metabase.query-permissions.core
```

Comment:
Could we include a comment with a heuristic explaining what should be found in this module, compared to the real permissions module, and maybe include why?